pgbouncer/manifest.json

```diff
@@ -10,6 +10,7 @@
     "linux",
     "mac_os"
   ],
+  "package_deps": false,
```

Comment:
We could bump it up, but the version check only looks at whether or not the mandatory fields are there